src/main/steps/applicant/task-list/tasklist-items.ts

```diff
@@ -1,4 +1,13 @@
 export const applicant_tasklist_items_en = {
   keep_your_details_private: 'Keep your details private',
   confirm_or_edit_your_contact_details: 'Confirm or edit your contact details',
+  your_needs_when_go_to_court: 'Your needs when you go to court',
+  application_submitted: 'Application submitted (FL401)',
+  witness_statement: 'Witness statement',
+  details_of_court_hearings: 'Check details of your court hearings',
+  orders: 'Orders',
+  upload_document: 'Upload a document',
+  see_all_documents: 'See all documents',
```

Comment:
Please re-verify all the taskList items title as per the prototype again. I see few mismatch there.